examples/Encrypted Inference-LinearRegression.ipynb

```diff
@@ -0,0 +1,1166 @@
+{
```

Comment:
<p>Q: Should we use timeit . https://stackoverflow.com/questions/17579357/time-time-vs-timeit-timeit</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/SyMPC/pull/276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Interesting will use it then.

Reply:
Unfortunately, I haven't been able to return output from the line I was timing using timeit module so I decided to go ahead with time module. The output of inference is required for calculating loss.

Reply:
Any particular error Hrishi, when using timeit module.